udata/core/dataset/api_fields.py

```diff
@@ -102,7 +103,8 @@
     'published': fields.ISODateTime(
         description='The resource publication date'),
     'last_modified': fields.ISODateTime(
-        attribute='modified', readonly=True,
+        attribute=lambda o: max([o.modified, to_naive_datetime(o.harvest.modified_at)]) if o.harvest and o.harvest.modified_at else o.modified,
```

Comment:
Could you add some test to check the result of the new computed date fields by API? :) :pray: